modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/data/handler/test/BatchEnginePortletDataHandlerTest.java

```diff
@@ -418,6 +447,33 @@ protected void doImportStagedModel(
 		}
 	}
 
+	@Test
+	@TestInfo("LPD-64361")
+	public void testExportImportPrivateLayoutsToOtherSite() throws Exception {
```

Comment:
I think if we need to test the filter of `externalReferenceCodes`, we should check somehow the file does not contain the layout2.
Maybe we could extend the test by creating a third layout, for example:
- Create a layout1, layout2, layout3
- Export layout1 and layout2
- Check only layout1 and layout2 have been exported
- Import only layout1
- Check layout2 and layout3 does not exist.